music21/spanner.py

```diff
@@ -1275,44 +1346,60 @@ def setPendingSpannedElementAssignment(
 
         Now set up su1 to get the next note assigned to it.
 
-        >>> sb.setPendingSpannedElementAssignment(su1, 'Note')
+        >>> sb.setPendingSpannedElementAssignment(su1, 'Note', 0.)
```

Comment:
little favor -- can you make these all 0.0, 1.0, etc. -- just a music21 style thing to always have a digit on both sides of a decimal point -- I think it makes it much clearer that we're working with floats not ints.

Reply:
Actually, I still need to understand why we had something that worked fine without 0.0 before and now needs a 0.0 to be passed to it.  I must be incredibly dense that I'm still not seeing the contribution here...

Reply:
I think you need to set up a scenario where the wrong spanned-element would be fried with the prior style of calling `freePendingSpannedElementAssignment` with a default `offset` and no `clientInfo`.   Right now all I see is extra work that people using spanner.py would have to do with no benefit.

Reply:
Also, there is no extra work for existing clients: these are optional params.  Existing clients will continue to work as before, with no changes.  I need to demonstrate that in the doctests.

Reply:
Floating point numbers all have a digit after the decimal point now.  And I have added scenarios and descriptions in the doctests.